app/uk/gov/hmrc/trackingconsentfrontend/assets/src/constants/cssClasses.ts

```diff
@@ -1,11 +1,18 @@
-export const COOKIE_BANNER_CLASS = 'cookie-banner'
-export const COOKIE_BANNER_QUESTION_CLASS = 'cookie-banner__question'
-export const COOKIE_BANNER_CONFIRMATION_CLASS = 'cookie-banner__confirmation-message'
+// Cookie banner related (uses cut-down version of Gov UK Frontend with prefixed class names)
+export const COOKIE_BANNER_CLASS = 'cbanner-cookie-banner'
+export const COOKIE_BANNER_QUESTION_CLASS = 'cbanner-cookie-banner__question'
+export const COOKIE_BANNER_CONFIRMATION_CLASS = 'cbanner-cookie-banner__confirmation-message'
+export const ACCEPT_ALL_CLASS = 'cbanner-cookie-banner__accept-all-button'
+export const HIDE_BUTTON_CLASS = 'cbanner-cookie-banner__hide-button'
+export const COOKIE_BANNER_GOV_UK_WIDTH_CONTAINER_CLASS = 'cbanner-govuk-width-container'
+
+// Cookie settings related (uses standard Gov UK Frontend)
+export const COOKIE_SETTINGS_NOTICE_CLASS = 'cookie-settings__notice'
 export const COOKIE_SETTINGS_CONFIRMATION_CLASS = 'cookie-settings__confirmation'
 export const COOKIE_SETTINGS_WRAPPER_CLASS = 'govuk-grid-column-two-thirds'
-export const ACCEPT_ALL_CLASS = 'acceptAll'
-export const HIDE_BUTTON_CLASS = 'cookie-banner__hide-button'
-export const GOV_UK_LINK = 'govuk-link'
-export const GOV_UK_BODY = 'govuk-body'
+
+// Gov UK frontend
+export const GOV_UK_BODY_CLASS = 'govuk-body'
```

Comment:
I'd assume these should be prefixed.

Reply:
These are the ones used on the cookie settings page, that are vanilla govuk-frontend.